compose/queue/api_data/taskqueue_v1beta2.py

```diff
@@ -0,0 +1,626 @@
+            The converted Acl object.
+        """
+        keys = [
+            "adminEmails", "consumerEmails", "producerEmails"
+        ]
+        if any(key not in json for key in keys):
+            raise Exception("Not all values specified.")
+        return Acl(**json)
+
+
+class TaskQueue(object):
+    """Describes a Google TaskQueue.
+
+    With methods for creation from json with error check and automatic
+    property building on creation.
+
+    TODO: Current implementation does not make use of stats and maxLeases etc.
+    """
+
+    def __init__(self, id=None, kind="taskqueues#taskqueue", maxLeases=None, stats=None, acl=None):
+        """Create a new TaskQueue object.
+
+        Args:
+            id: The identifier of the taskqueue, if None is given a
+                identifier is generated.
+            kind: The kind of the taskqueue.
+            maxLeases: The max number of leases an item can take before
+                it is removed from the queue.
+            stats: An Stats object recording information on the queue.
+            acl: An Acl object of Authorised users of the queue.
+        """
+        self.id = id if id is not None else generate_id()
+        self.kind = kind
+        self.maxLeases = maxLeases
+        self.stats = stats if stats is not None else Stats()
+        self.acl = acl if acl is not None else Acl()
+
+    def _asdict(self):
+        """Convert the current object into a dictionary.
+
+        Returns:
+            A dictionary of properties mapped to keys.
+        """
+        return {
+            "id": self.id,
+            "kind": self.kind,
+            "maxLeases": self.maxLeases,
+            "stats": self.stats,
+            "acl": self.acl
+        }
+
+    def to_json(self):
+        """Convert the current object into a json dictionary.
+
+        Returns:
+            A dictionary where the keys match the properties of the
+            object.
+        """
+        d = self._asdict()
+        d["stats"] = self.stats.to_json()
+        d["acl"] = self.acl.to_json()
+        return d
+
+    @staticmethod
+    def from_json(json):
+        """Create a new TaskQueue object from a json dictionary.
+
+        Includes error checking that all the required properties
+        are present.
+
+        Args:
+            json: A dictionary where keys map to Task object
+                properties.
+        Returns:
+            The converted TaskQueue object.
+        """
+        keys = [
+            "id", "kind", "maxLeases", "stats", "acl"
+        ]
+        if any(key not in json for key in keys):
+            raise Exception("Not all values specified.")
+
+        q = TaskQueue(**json)
+        q.stats = Stats.from_json(json["stats"])
+        q.acl = Acl.from_json(json["acl"])
+        return q
+
+
+# Taskqueue Blueprint
+taskqueue_v1beta2_api = Blueprint("taskqueue.v1beta2", __name__)
+
+
+@taskqueue_v1beta2_api.route(
+    "/<project>/taskqueues/<taskqueue>/tasks",
```

Comment:
Its a REST API url, this doesn't get deployed and is documented by Google there is no hiding it.